ui/apps/platform/src/Containers/Policies/Wizard/PolicyWizard.tsx

```diff
@@ -12,13 +20,24 @@ import { policiesBasePath } from 'routePaths';
 import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
 import { ExtendedPageAction } from 'utils/queryStringUtils';
 
+import {
+    POLICY_DEFINITION_DETAILS_ID,
+    POLICY_DEFINITION_LIFECYCLE_ID,
+    POLICY_DEFINITION_RULES_ID,
+    POLICY_BEHAVIOR_SCOPE_ID,
+    POLICY_BEHAVIOR_ENFORCEMENT_ID,
+    POLICY_BEHAVIOR_NOTIFIERS_ID,
+    POLICY_REVIEW_ID,
+} from '../policies.constants';
 import { getServerPolicy } from '../policies.utils';
 import { getValidationSchema } from './policyValidationSchemas';
 import PolicyDetailsForm from './Step1/PolicyDetailsForm';
 import PolicyBehaviorForm from './Step2/PolicyBehaviorForm';
 import PolicyCriteriaForm from './Step3/PolicyCriteriaForm';
 import PolicyScopeForm from './Step4/PolicyScopeForm';
-import ReviewPolicyForm from './Step5/ReviewPolicyForm';
+import PolicyEnforcementForm from './Step5/PolicyEnforcementForm';
+import AttachNotifiersForm from './Step6/AttachNotifiersForm';
```

Comment:
Observation, not necessarily change request, that updated design omits **Attach** from terminology.

Reply:
Good callout, went ahead and changed the name.